model/metadata.go

```diff
@@ -4,6 +4,7 @@ package model
 type Metadata struct {
 	Title       string   `json:"title"`
 	Description string   `json:"description"`
+	Survey      string   `json:"survey"`
```

Comment:
So I feel like this should actually be similar to how Keywords is but for some reason it didn't work?

Reply:
I think we only keep a single value for survey so this should be right 👍